### DISCUSSION TITLE
Multicore support

### DISCUSSION THREAD

**Discussion by inbum97** [mitmproxy/mitmproxy#5654] [Q&A] [answered]
Hi,
I want to know whether the product support multicore or multicpu. I came across this [post](https://discourse.mitmproxy.org/t/handeling-high-load-in-multicore-machine-with-in-line-scripting/51) back in 2016 where it stated that it was not supported at the moment. I am wondering if the product now support multicore in 2022. Thank you.

**Comment by mhils** [comment, marked as answer]
Everything stated in the post you linked is still correct. :)

**Comment by kuan121** [comment]
@mhils Just curious, did the mitmproxy team make any improvement in terms of performance since then? Two things I noticed are
1. @concurrent
2. `async` function

**Reply by mhils** [reply]
No, we didn't.
